### PR TITLE
Controller manual documentation. 

### PR DESCRIPTION
**Pull request by orenyodfat**

### CODE REVIEW

**Comment by orenyodfat** [comment]
> No need for h1 (# SOMETHING), this is already taken care of by the website 

Though if I would just want to browse the docs via github ?

**Comment by dev-matan-tsuberi** [comment]
Then make sure it's the same as the file name

**Comment by orenyodfat** [comment]
> Then make sure it's the same as the file name 

? e.g Avatar.md ? Avatar ?

**Comment by dev-matan-tsuberi** [comment]
Yes. Also Underscores are interpreted as space.
Do a `yarn docs serve` if you are unsure.

**Comment by orenyodfat** [comment]
Yes Avatar or Avatar.md ?

I prefer Avatar.

**Comment by dev-matan-tsuberi** [comment]
Avatar

File: docs/contracts/controller/DAOToken.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
-The *DAO token* is a simple token that is used by DAOs to assign value to, and reward participants actions. 
-It can be **minted** or **burned** by the DAO and also be **destroyed** in the event of an upgrade to a better token or in other circumstances.
\ No newline at end of file
+## DAOToken
+
+The *DAOToken* is an ERC20 token that can be used to assign value and reward DAOs members.
```
Comment (leviadam) [inline docs/contracts/controller/DAOToken.md]: Maybe ERC20 compatible?

File: docs/contracts/controller/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+A scheme can have any combination of the following permissions  :
+ - REGISTERED -  All registered schemes has this permission.
+                 Only registered schemes can perform controller operations.
+ - CAN_REGISTER - grant the scheme the permission to register other schemes.
+ - ADD_OR_REMOVE_GLOBAL_CONSTRAINT - grant the scheme the permission to add or remove a global constraint.
+ - CAN_UPGRADE - grant the scheme the permission to upgrade the controller.
+
+### Parameters
+
+The controller hold the hash of a parameters set for each scheme.
```
Comment (leviadam) [inline docs/contracts/controller/README.md]: holds?

Comment (orenyodfat) [reply docs/contracts/controller/README.md]: update at dc6fd98

File: docs/contracts/controller/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+This way a scheme can define a set of parameters which are specific for an organization(defined by the controller).
+
+## Global constraints
+
+A controller maintains and enforces a global constraints for the organization.
```
Comment (leviadam) [inline docs/contracts/controller/README.md]: enforces global constraints

File: docs/contracts/controller/Reputation.md [hunk 4]
```diff
@@ -1,2 +1,17 @@
-A *Reputation* is a way of assigning importance to participants and their votes. 
-*Reputation* is similar to regular tokens but with one crucial difference: **It is non-transferable**.
\ No newline at end of file
+## Reputation
+
+
+A *Reputation* is a way of assigning importance to participants and their votes.
```
Comment (leviadam) [inline docs/contracts/controller/Reputation.md]: Maybe influence instead of importance?

Comment (orenyodfat) [reply docs/contracts/controller/Reputation.md]: @leviadam how about : 
> Each DAO has Reputation System which allows peers to rate other peers in order to build trust . A reputation is use to assign influence measure to a DAO'S peers.
For example : A DAO might choose to use a reputation based voting mechainsim in order to take decisions. In this case a peer with relatively higher reputation value will have more influence in the organization.

File: docs/contracts/controller/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# CONTROLLER
```
Comment (dev-matan-tsuberi) [inline docs/contracts/controller/README.md]: should be `Controller`

**Comment by orenyodfat** [comment]
Thanks @dev-matan-tsuberi and @leviadam .

**Pull request merged**